src/BottomNavigation/BottomNavigationButton.js

```diff
@@ -131,7 +132,9 @@ export default class BottomNavigationButton extends Component {
         {...other}
         onClick={this.handleChange}
       >
-        {icon}
+        <div className={iconClassName}>
```

Comment:
> margin: 'auto',

Much better. I should have thought to try that. 😊 

I also took the liberty of adding some whitespace to the dense block of multi-line `const`s for readability. If that goes against code conventions I'll revert it, but I think it's for the better.